docs/walletkit/upgrade/from-web3wallet-ios.mdx

```diff
@@ -94,6 +94,21 @@ This led to an accumulation of stale pairings in wallets, resulting in degraded
 
 Pairings were never intended to be listed in the wallet, and wallets should only display active sessions to users.
 
+## WebSocket Connection Handling
+
+We've optimized the WebSocket connection management to improve performance and resource utilization. The SDK will now establish a WebSocket connection only when there's an explicit intention to send a request or subscribe to a topic. If none of these conditions are met, the WebSocket connection will remain closed by default.
+
+What's Changed?
+Previous Behavior: The SDK automatically initiated a WebSocket connection upon startup, regardless of active sessions or pending actions.
+New Behavior: The SDK delays establishing a WebSocket connection until it's necessary based on the app's activities.
```

Comment:
Needs an explicit newline otherwise these previous/new sentences won't be separated by line:
```suggestion

New Behavior: The SDK delays establishing a WebSocket connection until it's necessary based on the app's activities.
```